auth/src/main/java/org/aerogear/auth/utils/AuthStateManager.java

```diff
@@ -39,11 +39,11 @@ public OIDCCredentials read() {
      * Saves a token
      * @param authState token to be saved
      */
-    public synchronized void write(final OIDCCredentials authState) {
+    public synchronized void write(final OIDCCredentials authState) throws JSONException {
```

Comment:
I don't like the name "write".  I think save is more descriptive.  Also if we wrap the JSONExceptions above we aren't "leaking" it here.